app/containers/message/Components/CollapsibleQuote.test.tsx

```diff
@@ -0,0 +1,91 @@
+import { fireEvent, render, within } from '@testing-library/react-native';
+import React from 'react';
+
+import MessageContext from '../Context';
+import CollapsibleQuote from './CollapsibleQuote';
+
+// For some reason a general mock didn't work, I have to do a search
+jest.mock('react-native-mmkv-storage', () => ({
+	Loader: jest.fn().mockImplementation(() => ({
+		setProcessingMode: jest.fn().mockImplementation(() => ({
+			withEncryption: jest.fn().mockImplementation(() => ({
+				initialize: jest.fn()
+			}))
+		}))
+	})),
+	create: jest.fn(),
+	MODES: { MULTI_PROCESS: '' }
+}));
+
+const attachment = {
+	ts: '1970-01-01T00:00:00.000Z',
+	title: 'Engineering (9 today)',
+	fields: [
+		{
+			title: 'Out Today:\n',
+			value:
+				'Ricardo Mellu, 1 day, until Fri Mar 11\nLoma, 1 day, until Fri Mar 11\nAnitta, 3 hours\nDiego Carlitos, 19 days, until Fri Mar 11\nGabriel Vasconcelos, 5 days, until Fri Mar 11\nJorge Leite, 1 day, until Fri Mar 11\nKevin Aleman, 1 day, until Fri Mar 11\nPierre, 1 day, until Fri Mar 11\nTiago Evangelista Pinto, 1 day, until Fri Mar 11'
+		}
+	],
+	attachments: []
+};
+
+const mockFn = jest.fn();
+
+const Render = () => (
+	<MessageContext.Provider
+		value={{
+			onLongPress: () => {},
+			user: { username: 'Marcos' }
+		}}>
+		<CollapsibleQuote key={0} index={0} attachment={attachment} getCustomEmoji={mockFn} timeFormat='LT' theme='light' />
+	</MessageContext.Provider>
+);
+
+describe('CollapsibleQuote', () => {
```

Comment:
We should avoid using `testId` imo.
This is one mistake I made a lot while writing Detox tests.
It's a lot easier to write tests, of course, but that's not the reality.
This article is pretty good about it (: https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change